MELA/fortran/mod_TTBHiggs.F90

```diff
@@ -6,6 +6,7 @@ MODULE ModTTBHiggs
 public :: EvalXSec_PP_TTBH, EvalXSec_PP_BBBH, EvalAmp_GG_TTBH, EvalAmp_QQB_TTBH
 public :: InitProcess_TTBH
 public :: ExitProcess_TTBH
+public :: ubarSpi_Dirac, vSpi_Dirac
```

Comment:
Is this change necessary? If not, I would remove it.

Reply:
yes, it is necessary since the tHW Code uses the (massive) Dirac spinor subroutines from the ttbar code.

Reply:
Ok, but in this case, mod_TTBHiggs needs to be compiled before tHW, which cannot be parallelized. I would expect 'make -j' to fail when this module is included unless the dependency is included explicitly (and to be frank, I don't like too many dependencies across modules that do completely different ME calculations; we tried to move away from it already). Is it possible to duplicate these functions?

Reply:
counter-proposal: I could move the spinor functions into mod_TopDecay.F90
I would not duplicate too many functions since I already had problems with duplicated functions when implementing the code in JHUGen.

Reply:
Possible, but please make sure makefile works then.

Would you please also run the full compilation chain and do

cd test
python testME_all.py

and let me know if there are errors?

Reply:
> Ok, but in this case, mod_TTBHiggs needs to be compiled before tHW, which cannot be parallelized. I would expect 'make -j' to fail when this module is included unless the dependency is included explicitly (and to be frank, I don't like too many dependencies across modules that do completely different ME calculations; we tried to move away from it already). Is it possible to duplicate these functions?

since I have put "use ModTTBHiggs" in the ME routine which need the dirac spinor routines from ttbar, I would not expect the make -j to fail

Reply:
Doesn't matter, the makefile doesn't know what use statements the code has. It only cares about the dependencies specified.

Reply:
Understood.
In the makefile in the Generator directory I have actually specified the dependencies accordingly. 
I just realized that I did not push the respective makefile to MELA yet. I will also do that later.

Reply:
Good that we caught it then, thanks! Do you think you can also move the functions at the same time as you proposed?

Reply:
I have moved the dirac spinors to mod_TopDecay

Reply:
makefile doesn't need changing now

Reply:
> Possible, but please make sure makefile works then.
> 
> Would you please also run the full compilation chain and do
> 
> cd test
> python testME_all.py
> 
> and let me know if there are errors?

I cannot run the compilation chain for MELA here at home right now because I do not have ROOT installed on my laptop.
The exact same codes in the JHUGenerator directory did compile with the original makefile, though.

Reply:
I have tried to compile the MELA code on my office machine but I am still missing libraries:
libmcfm_707 and ljhugenmela
Can maybe one of you test the compilation chain?
Thanks